src/creds/creds.c

```diff
@@ -623,7 +624,7 @@ static int verb_decrypt(int argc, char **argv, void *userdata) {
                                 arg_tpm2_signature,
                                 arg_uid,
                                 &input,
-                                /* flags= */ 0,
+                                arg_allow_null?CREDENTIAL_ALLOW_NULL:0,
```

Comment:
```suggestion
                                arg_allow_null ? CREDENTIAL_ALLOW_NULL : 0,
```

Reply:
Also change ipc_decrypt_credential above?

Reply:
the varlink api does not pass through any flags.